hello-jni/README.md

```diff
@@ -0,0 +1,59 @@
+Hello JNI
+=========
+This sample show how to call Native C++ code from a Android Java activity using JNI. Note this require a preview build of Android Studio and the experimental version Android Gradle plugin, see the [Pre-requisites](#pre-requisites) and [Getting Started](#getting-started) instructions.
```

Comment:
nit: shows how to call Native C++ code from Android Java Activity using JNI. Note this requires
              ^                                                       ^                                                                             ^